### DISCUSSION TITLE
Creating Threads via discord.js master

### DISCUSSION THREAD

**Discussion by Flo4604** [discordjs/discord.js#6191] [Q&A] [answered]
Whenever I try to create a PUBLIC_THREAD I get the error code: BASE_TYPE_CHOICES", "Value must be one of (0, 2, 4, 6, 13, 14)."
Do i need to update my package or am I just doing something wrong

**Comment by vladfrangu** [comment, marked as answer]
1. You shouldn't use a master commit, use `@dev` instead if you want bleeding-edge releases!
2. Pretty sure the type is `GUILD_PUBLIC_THREAD` (see here: https://discord.js.org/#/docs/main/master/typedef/ChannelType)

**Reply by Flo4604** [reply]
I will install the dev branch and try with GUILD_PUBLIC_THREAD

**Reply by Flo4604** [reply]
Did not fix it, I am sending channel type 11 which should be the GUILD_PUBLIC_THREAD but it still errors. I can use them in the discord i am trying it that's why I am a bit confused.

**Reply by vladfrangu** [reply]
It errors *how*? What's the error now? Whats your code that causes this?

**Reply by Flo4604** [reply]
```
DiscordAPIError: Invalid Form Body
  type: Value must be one of (0, 2, 4, 6, 13, 14).
    method: 'post',
    path: '/guilds/guildId/channels',
    code: 50035,
    httpStatus: 400,
    requestData: {
      json: {
        name: 'new thread',
        topic: 'thread 1',
        type: 11,
        nsfw: undefined,
        bitrate: undefined,
        user_limit: undefined,
        parent_id: undefined,
        position: undefined,
        permission_overwrites: undefined,
        rate_limit_per_user: undefined
      },
      files: []
    }
  }
```

**Reply by vladfrangu** [reply]
That is not how you create a thread! Please read the documentation about them: https://discord.js.org/#/docs/main/master/class/TextChannel?scrollTo=threads, https://discord.js.org/#/docs/main/master/class/ThreadManager

**Reply by Flo4604** [reply]
My bad, next time I'll read the documentation more closely sorry for the inconvenience

**Reply by vladfrangu** [reply]
No worries! For further help, ask in our [Discord server](https://discord.gg/djs)
